### DISCUSSION TITLE
Consider adding Docker CLI to the SDK

### DISCUSSION THREAD

**Discussion by wizpresso-steve-cy-fan** [dotnet/dotnet-docker#4227] [General]
I need it because I want to publish the image in my CI containers that doesn't support multi-image layering: this means I'm constrained to using one image and will have to install Docker CLI tools all the time

**Comment by richlander** [comment]
Can you install it yourself? We haven't gotten any requests on this, so it doesn't seem like something we should add to the official image. Also (and this is probably the bigger issue) is that there is significant fragmentation on docker tools these days. If we add one CLI, someone is going to ask us to add another. We don't want to go down that path.
